README.md

```diff
@@ -12,6 +12,62 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not define multiple types (ie class, struct, enum...) in the same file; each type should have its own file.
```

Comment:
I think if you start to create multiple enums it may be worth creating a separate file what do you think? Makes sense for `i.e.` is it a standard that we have in the README?